### PR TITLE
[markeng-2223] [HOLD] dropdown POC

### PR DESCRIPTION
**Pull request by ch264**

### CODE REVIEW

**Review by jkonrath-postman — APPROVED** [review]
LGTM for a first POC. Will be great when it's on all the pages!

A few thoughts for the future:

* Maybe a little icon to the left of each item?
* All the descriptions need work (that's probably me)
* When on a small width device, the dropdown gets eaten off the right side. Is there some neat way to limit the width or make it responsive or whatever?

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -345,15 +351,60 @@ const Header = (props) => {
...
+                    <path d="M10.375,3.219,6,6.719l-4.375-3.5A1,1,0,1,0,.375,4.781l5,4a1,1,0,0,0,1.25,0l5-4a1,1,0,0,0-1.25-1.562Z" />
+                  </g>
+                </svg>
+              </a>
+              <div className="dropdown-menu">
+                <ul>
+                  <li>
+                    <a className="dropdown-item mb-3" href="/docs/getting-started/introduction/">
+                      <p className="dropdown-header h6" >Learning Center</p>
+                      <div className="dropdown-item dropdown-item-text-wrap">Learn how to use Postman. Check out the docs and support resources!</div>
```
Comment (jkonrath-postman) [inline src/components/Header/Header.jsx:381]:
```suggestion
                      <div className="dropdown-item dropdown-item-text-wrap">Learn about how to get started using Postman, and read more in the product docs.</div>
```

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -345,15 +351,60 @@ const Header = (props) => {
...
+                    </a>
+                  </li>
+                  <li>
+                    <a className="dropdown-item mb-3" href="https://learning.postman.com/labs/">
+                      <p className="dropdown-header h6" >Labs</p>
+                      <div className="dropdown-item dropdown-item-text-wrap">Flows, gRPC, WebSockets! Learn about Postman Labs and the cutting-edge features brewing in it.</div>
```
Comment (jkonrath-postman) [inline src/components/Header/Header.jsx:387]:
```suggestion
                      <div className="dropdown-item dropdown-item-text-wrap">Flows, gRPC, WebSockets! Learn about the latest cutting-edge features brewing in Postman Labs.</div>
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -345,15 +353,87 @@ const Header = (props) => {
...
+                >
+                  <g>
+                    <path d="M10.375,3.219,6,6.719l-4.375-3.5A1,1,0,1,0,.375,4.781l5,4a1,1,0,0,0,1.25,0l5-4a1,1,0,0,0-1.25-1.562Z" />
+                  </g>
+                </svg>
+              </a>
+              <div className="dropdown-menu lc-iconic">
+                <ul>
+                  <li>
+                    <a href="/docs/getting-started/introduction/" className="dropdown-item mb-3">
```
Comment (jkonrath-postman) [inline src/components/Header/Header.jsx:381]:
```suggestion
                    <a href="/docs/introduction/overview/" className="dropdown-item mb-3">
```

Comment (jkonrath-postman) [reply src/components/Header/Header.jsx:381]: You'd need to bring develop into this branch for that new URL to work

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Comment by ch264** [comment]
new PR https://github.com/postmanlabs/postman-docs/pull/4637

**Pull request closed without merging**